Makefile

```diff
@@ -145,7 +145,7 @@ else
 GO_LDFLAGS += -s
 endif
 
-all: unit-tests image-factory image-image-factory imager-base imager-tools integration.test integration tailwind lint
+all: unit-tests image-factory image-image-factory imager-base imager-tools integration.test integration update-to-talos-main integration-talos-main tailwind lint
```

Comment:
this is a nit, but probably we shouldn't have this in `all` (and some others as well?)

Reply:
i'll do a kres fix